src/Processors/Merges/ReplacingSortedTransform.h

```diff
@@ -18,7 +18,7 @@ class ReplacingSortedTransform final : public IMergingTransform<ReplacingSortedA
         WriteBuffer * out_row_sources_buf_ = nullptr,
         bool use_average_block_sizes = false)
         : IMergingTransform(
-            num_inputs, header, header, true,
+            num_inputs, header, header, true, false,
```

Comment:
C++ really needs named parameters.... we could use C++20 designated initializers and create a parameter struct, but it requires you to set every field, unlike C, so it's also ugly. I tend to write something like `/* expected_one_block = */ false`. clang-tidy even checks that these comment match the function declaration.